editor/plugins/spatial_editor_plugin.h

```diff
@@ -223,6 +223,10 @@ class SpatialEditorViewport : public Control {
 	Label *cinema_label;
 	Label *locked_label;
 
+	bool active;
```

Comment:
I have the feeling you are trying to replicate the focus functionality of the viewport here (has_focus(), grab_focus(), set_focus_mode())